Tests/FluentAssertions.Specs/Collections/Data/DataColumnCollectionAssertionExtensionsSpecs.cs

```diff
@@ -0,0 +1,433 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using FluentAssertions.Execution;
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections.Data
+{
+    public class DataColumnCollectionAssertionExtensionsSpecs
+    {
+        private DataColumn CreateTestDataColumn(int seed)
+        {
+            var column = new DataColumn("Column" + seed);
+
+            var random = new Random(seed);
+
+            column.DataType =
+                random.Next(4) switch
+                {
+                    0 => typeof(int),
+                    1 => typeof(string),
+                    2 => typeof(bool),
+                    _ => typeof(DateTime),
+                };
+
+            return column;
+        }
+
+        #region BeSameAs & NotBeSameAs
```

Comment:
❌ Please use nested classes for these. This will also allow you to keep the names of the tests shorter.